Formula/xcdiff.rb

```diff
@@ -0,0 +1,40 @@
+class Xcdiff < Formula
+  desc "Tool to diff xcodeproj files"
+  homepage "https://github.com/bloomberg/xcdiff"
+  url "https://github.com/bloomberg/xcdiff.git",
+    tag:      "0.10.0",
+    revision: "289872e572224ae429d0f4d25ff9be906c9df1a0"
+  license "Apache-2.0"
+  head "https://github.com/bloomberg/xcdiff.git", branch: "main"
+  depends_on :macos
+  depends_on xcode: "14.1"
+
+  resource "homebrew-testdata" do
+    url "https://github.com/bloomberg/xcdiff/archive/refs/tags/0.10.0.tar.gz"
+    sha256 "c093e128873f1bb2605b14bf9100c5ad7855be17b14f2cad36668153110b1265"
+  end
+
+  def install
+    system "make", "update_version"
+    system "make", "update_hash"
+    system "swift", "build", "--disable-sandbox", "--configuration", "release"
+    bin.install ".build/release/xcdiff"
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/xcdiff --version").chomp
+    resource("homebrew-testdata").stage do
+      assert_match "\n", shell_output(
+        "#{bin}/xcdiff -p1 Fixtures/ios_project_1/Project.xcodeproj -p2 Fixtures/ios_project_1/Project.xcodeproj -d",
+      )
+      assert_match "✅ BUILD_PHASES > \"Project\" target\n", shell_output(
+        "#{bin}/xcdiff " \
+        "-p1 Fixtures/ios_project_1/Project.xcodeproj " \
+        "-p2 Fixtures/ios_project_1/Project.xcodeproj " \
+        "-g BUILD_PHASES " \
+        "-t Project " \
+        "-v",
+      )
+    end
+  end
```

Comment:
Does this work?
```suggestion
    project = "Fixtures/ios_project_1/Project.xcodeproj"
    resource("homebrew-testdata").stage do
      assert_match "\n", shell_output("#{bin}/xcdiff -p1 #{project} -p2 #{project} -d")
      out = shell_output("#{bin}/xcdiff -p1 #{project} -p2 #{project} -g BUILD_PHASES -t Project -v")
      assert_match "✅ BUILD_PHASES > \"Project\" target\n", out
    end
  end
```
Otherwise maybe this?
```suggestion
    project = "Fixtures/ios_project_1/Project.xcodeproj"
    diff_args = "-p1 #{project} -p2 #{project}"
    resource("homebrew-testdata").stage do
      assert_match "\n", shell_output("#{bin}/xcdiff #{diff_args} -d")
      out = shell_output("#{bin}/xcdiff #{diff_args} -g BUILD_PHASES -t Project -v")
      assert_match "✅ BUILD_PHASES > \"Project\" target\n", out
    end
  end
```